scripts/extract-and-pull-request.sh

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
```

Comment:
Moved this to the action. There's no need to run the script locally, as the action can be manually triggered as needed, without setting up local environment.